docs/quickstart.md

```diff
@@ -101,8 +111,8 @@ Then run it with `bun run start`.
 
 ```bash
 $ bun run start
-  $ bun run index.ts
-  Listening on http://localhost:3000 ...
+$ bun run index.ts
+Listening on http://localhost:3000 ...
 ```
```

Comment:
<img width="790" height="165" alt="Image" src="https://github.com/user-attachments/assets/86fbdc01-5b25-4d85-bac8-791b0f47b87a" />

I'm pretty sure the indenting is intentional so that that its shown as nested, as the `bun run index.ts` is the script from package.json script and not what user specified in terminal